doc/appendix-de-kassensichv/reference-tables/reference-tables.md

```diff
@@ -279,3 +279,474 @@ For definitions regarding national laws, please refer to the appropriate appendi
+| `LOC_LAND` | Country | ISO 3166 ALPHA-3 county code | Mandatory, `dailyClosingMasterData.LocationCountry` |
+| `LOC_USTID` | VAT ID | String (15) | Mandatory, `dailyClosingMasterData.LocationVATId` |
+
+(`dailyClosingMasterData`is described above in the chapter "Master data module")
+
+##### File: Stamm_Kassen (cashregister.csv)
+
+| **Fieldname**            | **Description**          | **Format**          | **ft.input** |
+|----------------------|--------------------------|---------------------|---------------------|
+| `Z_KASSE_ID` | ID of the (closing) cashpoint | String (50) | `ftCashBoxIdentification` |
+| `Z_ERSTELLUNG` | Date of the cashpoint closing | ISO 8601 und RFC3339 date | From `cbReceiptMoment` of the daily closing receipt |
+| `Z_NR` | Nr. of the cashpoint closing | Integer | Automatically filled by ft |
+| `KASSE_BRAND` | Brand of the cash register | String (50) | Mandatory, `dailyClosingMasterData.CashRegisterBrand` |
+| `KASSE_MODELL` | Model designation | String (50) | Mandatory, `dailyClosingMasterData.CashRegisterModel` |
+| `KASSE_SERIENNR` | Serial number of the cash register | String (70) | `ftCashBoxID` ?(tbd) | 
+| `KASSE_SW_BRAND` | Brand name of the software | String (50) | Mandatory, `dailyClosingMasterData.CashRegisterSWBrand` |
+| `KASSE_SW_VERSION` | Version of the software | String (50) | Mandatory, `dailyClosingMasterData.CashRegisterSWVersion`  |
+| `KASSE_BASISWAEH_CODE` | Basis currency of the cash register | ISO 4217 currency code | Mandatory, `dailyClosingMasterData.CashRegisterBaseCurrency`  |
+| `KEINE_UST_ZUORDNUNG` | VAT not determinable | 0 or 1 | Automatically filled by ft |
+
+(`dailyClosingMasterData`is described above in the chapter "Master data module")
+
+##### File: Stamm_Terminals (slaves.csv)
+
+| **Fieldname**            | **Description**          | **Format**          | **ft.input** |
+|----------------------|--------------------------|---------------------|---------------------|
+| `Z_KASSE_ID` | ID of the (closing) cashpoint | String (50) | `ftCashBoxIdentification` |
+| `Z_ERSTELLUNG` | Date of the cashpoint closing | ISO 8601 und RFC3339 date | From `cbReceiptMoment` of the daily closing receipt |
+| `Z_NR` | Nr. of the cashpoint closing | Integer | automatically filled by ft |
+| `TERMINAL_ID` | ID of the terminal | String (50) |Mandatory, `dailyClosingMasterData.Terminals.Id` |
+| `TERMINAL_BRAND` | Brand of the terminal | String (50) | Mandatory, `dailyClosingMasterData.Terminals.Brand`|
+| `TERMINAL_MODELL` | Model designation | String (50) | Mandatory, `dailyClosingMasterData.Terminals.Model` |
+| `TERMINAL_SERIENNR` | Serial number of the terminal | String (70) | Mandatory, `dailyClosingMasterData.Terminals.SerialNr` |
+| `TERMINAL_SW_BRAND` | Brand name of the software | String (50) | Mandatory, `dailyClosingMasterData.Terminals.SWBrand` |
+| `TERMINAL_SW_VERSION` | Version of the software | String (50) | Mandatory, `dailyClosingMasterData.Terminals.SWVersion` |
```

Comment:
Add the data to the ft.Portal and synchronize it?
This could be needed for the registration-(process) to the Finanzamt.

Reply:
yes, thank you, we are thinking about how to provide the master data